admin/javascript/LeftAndMain.EditForm.js

```diff
@@ -77,23 +85,44 @@
 					}
 				}
 
+				// Reset error display
+				this.setValidationErrorShown(false);
+
 				// TODO
 				// // Rewrite # links
 				// html = html.replace(/(<a[^>]+href *= *")#/g, '$1' + window.location.href.replace(/#.*$/,'') + '#');
 				// 
 				// // Rewrite iframe links (for IE)
 				// html = html.replace(/(<iframe[^>]*src=")([^"]+)("[^>]*>)/g, '$1' + $('base').attr('href') + '$2$3');
-				
-				// Show validation errors if necessary
-				if(this.hasClass('validationerror')) {
-					// Ensure the first validation error is visible
-					var tabError = this.find('.message.validation, .message.required').first().closest('.tab');
-					$('.cms-container').clearCurrentTabState(); // clear state to avoid override later on
-					tabError.closest('.ss-tabset').tabs('option', 'active', tabError.index('.tab'));
-				}
 
 				this._super();
 			},
+			'from .cms-tabset': {
```

Comment:
Interesting and I suspected as much. One thing I'm still unsure of: 

Can you decompose the meaning of this object literal key, for me? What significance does the `from` have here in conjunction with the (apparent) `.cms-tabset` selector? Or is this key just a selector? The context of it makes sense to me if:
- It executes with the `this` of any closure defined as a value within this object literal bound to the outer selected `.cms-edit-form`, similar to the above key `onadd`
- The `from` Prefix is magic, similar to `on` since typically definitions here are methods. Instead, `on` converts the closure to an event listener and `from` contorts the context of child keys, with context then reverting back. 

Is that correct! Seems useful... But complicated to explain, no wonder it's undocumented 😂